
package uy.com.group05.baascore.sl.services.soap;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 2.4.6
 * 2013-11-28T05:38:37.667-02:00
 * Generated source version: 2.4.6
 */

@WebFault(name = "UserCantAccessAppException", targetNamespace = "http://soap.services.sl.baascore.group05.com.uy/")
public class UserCantAccessAppException_Exception extends Exception {
    
    private uy.com.group05.baascore.sl.services.soap.UserCantAccessAppException userCantAccessAppException;

    public UserCantAccessAppException_Exception() {
        super();
    }
    
    public UserCantAccessAppException_Exception(String message) {
        super(message);
    }
    
    public UserCantAccessAppException_Exception(String message, Throwable cause) {
        super(message, cause);
    }

    public UserCantAccessAppException_Exception(String message, uy.com.group05.baascore.sl.services.soap.UserCantAccessAppException userCantAccessAppException) {
        super(message);
        this.userCantAccessAppException = userCantAccessAppException;
    }

    public UserCantAccessAppException_Exception(String message, uy.com.group05.baascore.sl.services.soap.UserCantAccessAppException userCantAccessAppException, Throwable cause) {
        super(message, cause);
        this.userCantAccessAppException = userCantAccessAppException;
    }

    public uy.com.group05.baascore.sl.services.soap.UserCantAccessAppException getFaultInfo() {
        return this.userCantAccessAppException;
    }
}
